libraries/standard/serializer/src/cbor/iot_serializer_tinycbor_decoder.c

```diff
@@ -78,6 +103,22 @@ typedef struct _cborValueWrapper
 
 /*-----------------------------------------------------------*/
 
+static size_t calculateSizeOfCborObject( CborValue * pValue )
+{
+    IotSerializer_Assert( pValue != NULL );
+    CborValue nextValue;
+
+    /* Copy the contents of the current CBOR object so that we can advance to the next object. */
+    memcpy( &nextValue, pValue, sizeof( CborValue ) );
+
+    IotSerializer_Assert( cbor_value_advance( &nextValue ) == IOT_SERIALIZER_SUCCESS );
```

Comment:
Better to make it separate line for easy debugging.

Reply:
Good point. Will do so.